loops.js

```diff
@@ -1,4 +1,4 @@
-// Begin status code
+// Begin status code (aangepast)
 
 // ==========================================
 // Opdracht 1. Maak een for-loop die 3 keer "Hoera!" in de terminal logt
```

Comment:
Helemaal goed

Reply:
Oef 2 / Klopt, zo kan het ook inderdaad. Had ik zelf niet aan gedacht.

Oef 3 / Doet het.  Alleen je bent hier en op meer plekken de brackets { } vergeten bij het if-statement. 

Ook een kleine opmerking: bij de comments staat "16x dupliceren", dat klinkt als "een kopie maken" terwijl het gaat om "optellen". Wel goed dat je comments plaatst.

Oef 4 / Kloppende code. Leuk dat je de 'i' een eigen naam heb gegeven. Ik zou dit dan echter alleen doen als het iets toevoegt en meer context geeft aan de code. Nu gebruik je bijvoorbeeld "biggerThen" maar in de code moet "biggerThen" juist KLEINER zijn dan 3 en dan is de naam een beetje verwarrend. Of was dat juist de grap misschien :D Persoonlijk had ik in dit geval "number" een betere naam gevonden, want dan lees je de code als "if number is smaller than three, print number" en niet "if bigger then is smaller than three, print bigger then".